systems-ui/src/pages/system-detail/SystemDetail.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component} from 'react';
+import SystemsOrchestrator from "../../common/components/systems/systems-orchestrator/SystemsOrchestrator";
+
+class SystemDetail extends Component {
```

Comment:
I think this can be made stateless.

Reply:
Same as above, I will change this to a PureComponent as soon I have some time :)